cfgov/v1/models/base.py

```diff
@@ -131,17 +131,6 @@ class CFGOVPage(Page):
         ObjectList(settings_panels, heading='Configuration'),
     ])
 
-    def get_authors(self):
-        """ Returns a sorted list of authors. Default is alphabetical """
-        return self.alphabetize_authors()
-
-    def alphabetize_authors(self):
-        """ Alphabetize authors of this page by last name, then first name if needed """
-        # First sort by first name
-        author_names = sorted(self.authors.names())
-        # Then sort by last name
-        return sorted(author_names, key=lambda x: x.split()[-1])
```

Comment:
In reference to https://github.com/cfpb/cfgov-refresh/pull/2384/files#r78132526, I'm not sure this does exactly what you're intending unless I'm making an incorrect assumption:
```python
In [3]: names = ['Barry White', 'Alex Trebek', 'Ben Hilsen']

In [4]: snames = sorted(names)

In [5]: snames
Out[5]: ['Alex Trebek', 'Barry White', 'Ben Hilsen']

In [6]: slnames = sorted(names, key=lambda x: x.split()[-1])

In [7]: slnames
Out[7]: ['Ben Hilsen', 'Alex Trebek', 'Barry White']
```
Since you're sorting twice, the second `sorted` call is overwriting the first. Is this what was intended? If yes, why not sort the first time?

To me, sorting alphabetically by the whole name should be fine which would leave us without the need for these methods; unless there are other requirements.